internal/gcsx/append_object_creator_test.go

```diff
@@ -240,7 +240,7 @@ func (t *AppendObjectCreatorTest) CallsComposeObjectsWithObjectProperties() {
 		WillOnce(Return(nil))
 
 	// Call
-	t.call()
+	_, _ = t.call()
```

Comment:
Should we capture err and assert?

Reply:
Asserting for error not being nil causes tests to fail.

Reply:
Is err not nil?

Reply:
No, it isn't